py-polars/polars/interchange/dataframe.py

```diff
@@ -22,7 +22,7 @@ class PolarsDataFrame(InterchangeDataFrame):
     Parameters
     ----------
     column
-        The Polars DataFrame backing the dataframe object.
+        The Polars DataFrame backing the dataframe interchange object.
```

Comment:
Could this please just remain "dataframe object"? Matches line 20.

```suggestion
        The Polars DataFrame backing the dataframe object.
```